developing-with-classicpress/tools-and-standards

```diff
@@ -0,0 +1,14 @@
+# Tools and Standards
+
+## Recommended Tools for Development
+
+- [PHPStorm](https://www.jetbrains.com/phpstorm/)
+- [SublimeText](https://www.sublimetext.com)
```

Comment:
I am not sure about sharing code editors, there are a lot and I don't really want to get into the game of having to update this with every single one. Thoughts?

Reply:
Valid point.